### PR TITLE
Page Fixes and New Background Color Prop

### PR DESCRIPTION
**Pull request by morethanfire**
This work is not be not breaking. I consider it a minor version to be able to sync up with design and be able to fix the problem in HC before it grows to large. 

However with that being said, it will cause a small amount of regressions due to how some of the Page sub components were being styled. I went through HC and found only 10 files that the Page component is being used and out of those ten there are only 5 locations that a regression would be caused. I packed these changes and installed on HC and fixes those 5 locations. That branch is now wait for this work to be released sot hat we can submit a PR in HC.

### CODE REVIEW

File: src/templates/content.scss [hunk 1]
```diff
@@ -18,6 +24,15 @@ $prefixPageContent: page--content;
     }
 }
 
+@media only screen and (min-width: 508px) {
```
Comment (groberts314) [inline src/templates/content.scss]: NAB: Should we get the standardized breakpoints mixin setup in React CM UI as well?

Comment (morethanfire) [reply src/templates/content.scss]: I do think the mixins should come from this package, but for now I think we are okay, right?

Comment (groberts314) [reply src/templates/content.scss]: Right you are; hence NAB :-)

Comment (morethanfire) [reply src/templates/content.scss]: Yep, yep. Thanks!

**Pull request merged**